src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -144,7 +144,7 @@
     private boolean mSyncInProgress = false;
 
     private static String DIALOG_UNTRUSTED_CERT = "DIALOG_UNTRUSTED_CERT";
-    private static String DIALOG_CREATE_FOLDER = "DIALOG_CREATE_FOLDER";
+    public static String DIALOG_CREATE_FOLDER = "DIALOG_CREATE_FOLDER";
```

Comment:
I slightly refactored the code.
* the dialog isn't used in the Activity anymore
  - so I removed the method in the activity https://github.com/owncloud/android/commit/8d038e89a9560e190f282d5b074b909193292822
  - moved the requestCode to the fragment https://github.com/owncloud/android/commit/f34a9ddf5b2c2a5bb326fc08ddac65c3e3546737
  - also removed the "other" action implementations since they are now handeled by the fab implementation in the fragment https://github.com/owncloud/android/commit/7e82baa8bc068bee956f6f03b61c06151c953eac
* shortened the upload call by creating a static helper method within the upload activity https://github.com/owncloud/android/commit/7e82baa8bc068bee956f6f03b61c06151c953eac